src/client/theme-default/components/VPAlgoliaSearchBox.vue

```diff
@@ -85,14 +85,6 @@ function initialize(userOptions: DefaultTheme.AlgoliaSearchOptions) {
             if (route.path === relativeHit) {
               return
             }
-
-            // if the hits goes to another page, we prevent the native link
-            // behavior to leverage the Router loading feature.
-            if (route.path !== relativeHit) {
-              event.preventDefault()
-            }
-
-            router.go(relativeHit)
```

Comment:
If we are removing these then there is no need to provide the `onClick` prop, there is nothing being done there except for this.

If I am correct, then we can remove that safely as this https://github.com/vuejs/vitepress/blob/main/src/client/app/router.ts#L129 part of the code handles all the links.